src/main/java/me/svdev/mscategory/service/CategoryService.java

```diff
@@ -0,0 +1,39 @@
+package me.svdev.mscategory.service;
+
+import lombok.RequiredArgsConstructor;
+import me.svdev.mscategory.dao.entity.Category;
+import me.svdev.mscategory.dao.repository.CategoryRepository;
+import me.svdev.mscategory.mapper.CategoryMapper;
+import me.svdev.mscategory.model.response.CategoriesResponse;
+import me.svdev.mscategory.model.dto.CategoryDto;
+import org.springframework.stereotype.Service;
+
+import static me.svdev.mscategory.mapper.CategoryMapper.mapEntityToDto;
+
+@RequiredArgsConstructor
+@Service
+public class CategoryService {
+
+    private final CategoryRepository categoryRepository;
+
+    public CategoriesResponse getCategories() {
+        return new CategoriesResponse(categoryRepository.findAll().stream().map(CategoryMapper::mapEntityToDto).toList());
```

Comment:
if your category isVisible=false, you return this. this is bug, please don't return isVisible = false categories